_data/surajsahani/surajsahani.md

```diff
@@ -0,0 +1,7 @@
+---
+name: SURAJ SAHANI
+institution: Rawal Institute Of Engineering And Technology
+profile_pic: surajsahani.png
```

Comment:
The image path name seems incorrect, you might want to change `surajsahani.png` to `surajsahani.jpg`

Reply:
Thanks, mitulsavani I have to change the file requested can you review again. thank you.